expresssession/server/server.js

```diff
@@ -27,22 +33,77 @@ const getindex = (time) => {
 const app = express();
 
 
+app.use(session({ secret: "sshhhhss", saveUninitialized: true, resave: true, cookie: { maxAge: (60000 * 5) } }))
 
 app.use(cors())
 app.use(bodyParser.json())
 app.use(bodyParser.urlencoded({ extended: false }));
+app.use('/aboutus', express.static(__dirname + "/aboutus"))
+
+
+
+const sessionValidator = (req, res, next) => {
+    console.log("this happened")
+    const sessionid = req.params.id;
+    for (i in sessions) {
+        if (sessions[i].sessionid == sessionid) {
+            if (sessions[i].expire < Date.now())
+                req.valid = false;
+            else { req.valid = true; }
+
+            next()
+            console.log("this reaches here")
+        }
+    }
+    req.valid = false;
+    next();
+}
+
+
 
-app.use( '/aboutus' , express.static(__dirname + "/aboutus"))
 
-app.get('/', (req, res) => {
-   
-    res.send(data)
+app.get('/session', (req, res) => {
+    console.log(req.session)
+
+    let sess = req.session;
+    res.send(sess);
+})
+
+
+
+app.post('/session', (req, res) => {
+
+    const sessionObj = {
+        sessionid: req.sessionID,
+        expire: Date.now() + (60000)
+    }
+    sessions.push(sessionObj);
```

Comment:
Why are we pushing in sessions?

Reply:
it was suppose to stores multiple session users .